docs/supported_docker_environment/windows.md

```diff
@@ -23,6 +23,12 @@ effort.
 
 * WCOW is currently not supported, since Testcontainers uses auxiliary Linux containers for certain tasks and Docker for Windows does not support hybrid engine mode at the time of writing.
 
+## Docker for Windows with WSL2 backend
+
+WSL2 should work out of the box.
+
+However, we have observed some problems using Docker for Windows with WSL2 backend with regards to very old images from the Docker Hub that would not start, so some caution when upgrading might be in order.
```

Comment:
I think we should be clearer to give people confidence: either we should work out why these old images aren't working and declare a specific limitation, or tell people what to look out for (or both!)

We should probably also attach a 'beta' label to WSL2 compatibility.

Reply:
Yep, both sounds good. 
I am also confident that we might fight a corresponding issue in Docker or WSL repo somewhere that we can reference.

Reply:
See the last comment for the reason of the images failing. I will update docs accordingly.